### DISCUSSION TITLE
Help : Which all domains should I whitelist to make sure github coplot works well behind the org proxy

### DISCUSSION THREAD

**Discussion by Rahulbuk** [community/community#114454] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My developer workstation is behind my organization  proxy and access to the internet is very limited.
Can someone help me list down the domains that I should whitelist so that Github Copilot works well. 
So far I have captured 

- github.com
- githubcopilot.com
- githubassets.com
- githubusercontent.com

**Comment by Akash1134** [comment]
Hi @Rahulbuk ,

We have a comprehensive documentation to help your setup Copilot in your Organization: [Configuring proxy settings for GitHub Copilot](https://docs.github.com/en/enterprise-cloud@latest/copilot/configuring-github-copilot/configuring-network-settings-for-github-copilot?tool=jetbrains#configuring-proxy-settings-for-github-copilot)

Hope that helps!

**Reply by Rahulbuk** [reply]
Thanks for the replying but I am looking for all the domains that I should whitelist in proxy server itself. Configuring proxy in IDE is not a big deal and is quite clear to me how to do it

**Comment by davecheney** [comment, marked as answer]
@Rahulbuk the required URLs are documented here https://docs.github.com/en/enterprise-cloud@latest/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot

**Reply by Rahulbuk** [reply]
Super this is what i was looking for.
Many thanks
